packages/base-styles/_variables.scss

```
@@ -150,4 +182,4 @@ $block-selected-margin: 3px;
 $block-selected-border-width: 1px;
 $block-selected-padding: 0;
 $block-selected-child-margin: 5px;
-$block-selected-to-content: $block-edge-to-content - $block-selected-margin - $block-selected-border-width;
+$block-selected-to-content: $block-edge-to-content - $block-selected-margin - $block-selected-border-width;
```

Same re. not removing the last line

```suggestion
$block-selected-to-content: $block-edge-to-content - $block-selected-margin - $block-selected-border-width;

```